--- vstudio.py.orig	2021-06-22 08:07:48.732917001 +0000
+++ vstudio.py	2021-06-22 08:09:33.370637468 +0000
@@ -532,7 +532,7 @@
         self.descriptionDone = 'built ' + self.describe_project()
         yield self.updateSummary()
 
-        command = (('FOR /F "tokens=*" %%I in (\'vswhere.exe -property  installationPath\')" '
+        command = (('FOR /F "tokens=*" %%I in (\'vswhere.exe -products * -property  installationPath\')" '
                     ' do "%%I\\%VCENV_BAT%" x86 && msbuild "{}" /p:Configuration="{}" '
                     '/p:Platform="{}" /maxcpucount').format(self.projectfile, self.config,
                                                             self.platform))
